### ISSUE TITLE
Suggest usable file name when saving log file somewhere

### ISSUE THREAD

**Issue by oliver** [darshanparajuli/LogcatReader#25]
TLDR: when saving a "saved log" somewhere else, an empty file name is suggested as new file name (rather than eg. the "logcat_timestamp.txt" name).

Long description: I saved the current log with Menu -> Save, then went to "Saved Logs", long-pressed on the new entry, and hit the "disk" icon. A save dialog appeared, with a text entry field at the bottom (next to the "Save" button). The text entry field was empty. When hitting Save, the log was saved in a file named "(invalid).txt".

I would suggest to pre-fill the text field with the log file name shown in the Saved Logs list instead.

This happened with Logcat Reader 1.5.4 from F-Droid under Android 7.1.2 (LineageOS 14.1).

**Comment by darshanparajuli** [comment]
Nice catch and a great suggestion. It'll be included on the next update shortly.

**Comment by darshanparajuli** [comment]
e24ef0bd3d67f69ed5f02998eb3f689fcb88b9ad
